internal/testutils/networking/iptables.go

```diff
@@ -50,3 +55,25 @@ func FirewalldRunning() bool {
 	state, err := exec.Command("firewall-cmd", "--state").CombinedOutput()
 	return err == nil && strings.TrimSpace(string(state)) == "running"
 }
+
+// FirewalldReload reloads firewalld and waits for the daemon to re-create its rules.
+// It's a no-op if firewalld is not running, and the test fails if the reload does
+// not complete.
+func FirewalldReload(t *testing.T, d *daemon.Daemon) {
+	t.Helper()
+	if !FirewalldRunning() {
+		return
+	}
+	lastReload := d.FirewallReloadedAt(t)
+	res := icmd.RunCommand("firewall-cmd", "--reload")
+	assert.NilError(t, res.Error)
+
+	poll.WaitOn(t, func(_ poll.LogT) poll.Result {
+		latestReload := d.FirewallReloadedAt(t)
+		if latestReload != "" && latestReload != lastReload {
+			t.Log("Firewalld reload completed at", latestReload)
+			return poll.Success()
+		}
+		return poll.Continue("firewalld reload not complete")
+	}, poll.WithDelay(100*time.Millisecond), poll.WithTimeout(20*time.Second))
```

Comment:
FWIW; default is 100 miliseconds interval (so we don't need to set it), and 10 seconds timeout. If 10 seconds would still be enough for this one, we could remove this config.

(not a blocker for sure; mostly that I removed a bunch of these at some point where the custom options were the same as the default)